src/main/java/org/broadinstitute/hellbender/engine/FeatureInput.java

```diff
@@ -71,6 +73,39 @@
         Utils.nonNull(rawArgumentValue, "rawArgumentValue");
     }
 
+    /**
+     * Construct a FeatureInput from a GATKPath argument value.
+     *
+     * @param gatkPath an existing GATKPath object
+     *
+     * This constructor is meant to be called only by the engine and test classes,
+     * which is why it has package access.
```

Comment:
Comment is inaccurate -- this constructor is public, not package access

Reply:
Fixed.